warehouse/query-core/src/main/java/datawave/query/postprocessing/tf/TermOffsetPopulator.java

```diff
@@ -85,17 +85,17 @@ public Document document() {
         Map<String,Object> map = new HashMap<>();
         Map<String,TermFrequencyList> termOffsetMap = Maps.newHashMap();
         
-        Map<String,TermFrequencyList> termOffsetMap1 = (Map<String,TermFrequencyList>) (map1.get(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME));
-        Map<String,TermFrequencyList> termOffsetMap2 = (Map<String,TermFrequencyList>) (map2.get(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME));
+        TermOffsetMap termOffsetMap1 = (TermOffsetMap) map1.get(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME);
+        TermOffsetMap termOffsetMap2 = (TermOffsetMap) (map2.get(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME));
         
         if (termOffsetMap1 == null) {
```

Comment:
Maybe this routine could be achieved by moving the termfrequency merge into the TermOffsetClass as a mergeTermFrequencies(TermOffsetMap) method or something like that.

Reply:
Should be straightforward to do, I'll make the change.